src/seedu/addressbook/AddressBook.java

```diff
@@ -250,17 +229,13 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
-            showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
-            exitProgram();
-        }
-
-        if (args.length == 1) {
-            setupGivenFileForStorage(args[0]);
-        }
-
-        if(args.length == 0) {
+    	if (args.length == 0) {
             setupDefaultFileForStorage();
+        } else if (args.length == 1) {
+        	setupGivenFileForStorage(args[0]);
```

Comment:
indentation is off here. we require you to use 4 spaces for indentation.
Tip: you can get eclipse to do this for you, check out how!